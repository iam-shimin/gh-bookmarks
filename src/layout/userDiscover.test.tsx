import React from 'react';
import renderWithStoreAndRouter from 'utils/render';
import UserDiscover from './userDiscover';

describe('<UserDiscoverModal />', () => {
	test('match snapshot', () => {
		const { asFragment } = renderWithStoreAndRouter(<UserDiscover />);
		expect(asFragment()).toMatchSnapshot();
	});
});