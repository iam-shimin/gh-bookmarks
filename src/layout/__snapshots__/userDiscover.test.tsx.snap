// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserDiscoverModal /> match snapshot 1`] = `<DocumentFragment />`;
